libpromises/evalfunction.c

```diff
@@ -7388,7 +7388,7 @@ void ModuleProtocol(EvalContext *ctx, char *command, const char *line, int print
             else if (StringMatchFullWithPrecompiledRegex(context_name_rx, content))
             {
                 Log(LOG_LEVEL_VERBOSE, "Module changed variable context from '%s' to '%s'", context, content);
-                strcpy(context, content);
+                strlcpy(context, content, context_size);
```

Comment:
'filename' is based on a parameter 'command', how do you know it's less than 50 chars?
Perhaps you did not see that one..?

For the second strlcpy() of context in this function it looks to be safe to make sure it's larger
than 50 chars and use strcpy(), but what if someone changes the function in the future?
Is it really worth risking?

I think we should keep strlcpy() here, why are you so insistent on strcpy()?

I added a ProgrammingError() in case < 51 char buffer is passed, though.